timesketch/api/v1/resources.py

```diff
@@ -1166,3 +1169,55 @@ def delete(self, sketch_id, timeline_id):
         sketch.timelines.remove(timeline)
         db_session.commit()
         return HTTP_STATUS_CODE_OK
+
+class SearchIndexResource(ResourceMixin, Resource):
```

Comment:
This is great, just drop the API key. We should also create a IndexResource using es.create_index to create elastic indes via the API